api/routes/v1/units.company.js

```diff
@@ -0,0 +1,81 @@
+const router = require('express').Router({mergeParams: true})
+const Units = require('../../models/units')
+const Companies = require('../../models/companies')
+const Employees = require('../../models/employees')
+
+// Units
+router.patch('/', async (req, res, next) => {
+  const status = 201
+  const response = await Units.findByIdAndUpdate(req.params.unitId, {$set:{company:req.body.company}})
+  res.json({status, response})
+})
+
+router.delete('/', async (req, res, next) => {
+  const status = 201
+  const response = await Units.findByIdAndUpdate(req.params.unitId, {$unset:{company:""}})
+  res.json({status, response})
+})
+
+router.get('/employees', async (req, res, next) => {
+  const status = 200
+  const Unit = await Units.findById(req.params.unitId).populate({
```

Comment:
Careful assigning this to an uppercase `Unit` variable. The convention here is to use this for Models, not necessarily the data that comes back from a model.